templates/service-template.yml

```diff
@@ -625,6 +642,15 @@ objects:
     data:
       node-prewarming-configuration.yaml: |-
         ${NODE_PREWARMING_CONFIG}
+  - kind: ConfigMap
+    apiVersion: v1
+    metadata:
+      name: kas-fleet-manager-admin-authz-config
+      annotations:
+        qontract.recycle: "true"
+    data:
+      admin-auth-configuration.yaml: |-
```

Comment:
Shouldn't this be `admin-authz-configuration.yaml`?

Reply:
thanks. fixed